### ISSUE TITLE
Does the wiring of force mapping support flow effects, are there configurable fields, or are there other compatible solutions

### ISSUE THREAD

**Issue by wty123** [visjs/vis-network#967]
Does the wiring of force mapping support flow effects, are there configurable fields, or are there other compatible solutions

**Comment by Thomaash** [comment]
Hi @wty123,

I'm sorry, but I have no idea what you're asking. Could you maybe dumb it down a bit?

Thanks.

**Comment by wty123** [comment]
Hi  @Thomaash

The edges in my current relationship graph need to be dynamic, similar to flowing，I want to know if this effect can be achieved with css3 or gif，Or there are other ways to achieve this effect

Thanks
